Makefile

```diff
@@ -159,14 +159,6 @@ docker/all: 	                                ## Build, migrate, loaddata, transl
 	make docker/loaddata
 	make docker/translations
 
-.PHONY: docker/db_snapshot
-docker/db_snapshot:   ## Snapshot database
-	docker exec galaxy_ng_postgres_1 pg_dump -U galaxy_ng -F c -b -f "/galaxy.backup" galaxy_ng
-
-.PHONY: docker/db_restore
-docker/db_restore:   ## Restore database from last snapshot
-	docker exec galaxy_ng_postgres_1 pg_restore --clean -U galaxy_ng -d galaxy_ng "/galaxy.backup"
-
```

Comment:
Commit message says "Fix Makefile". Is there any elaboration? Curious what it broke and how.  I don't think we need this in the Makefile, so no issue with removing it, just curious.

Reply:
I created these commands to help debug the migration and accidentally merged them with the migration PR. I added them separately to master with some better support here (https://github.com/ansible/galaxy_ng/pull/1281), and they ones from master were causing conflicts.